src/components/ColorPicker/tests/ColorPicker.test.tsx

```diff
@@ -91,46 +124,50 @@ describe('<ColorPicker />', () => {
 
     it('passes false to passive prop for "mousemove" EventListener when dragging', () => {
       const onChangeSpy = jest.fn();
-      const colorPicker = mountWithAppProvider(
+      const colorPicker = mountWithApp(
         <ColorPicker color={red} onChange={onChangeSpy} />,
       );
 
-      colorPicker
-        .find(Slidable)
-        .at(SlidableType.BrightnessSaturation)
-        .simulate('mousedown');
-
-      const mouseMoveListener = colorPicker
-        .find(EventListener)
-        .filterWhere((listener) => listener.prop('event') === 'mousemove');
+      colorPicker!.find(Slidable)!.find('div')!.trigger('onMouseDown', {
```

Comment:
No need for that first non-null assertion as colorPicker will never be null

```suggestion
      colorPicker.find(Slidable)!.find('div')!.trigger('onMouseDown', {
```